phpmyfaq/src/phpMyFAQ/Category.php

```diff
@@ -103,9 +103,10 @@ class Category
      */
     public function __construct(
         private readonly Configuration $configuration,
-        array $groups = [],
-        bool $withPerm = true
-    ) {
+        array                          $groups = [],
```

Comment:
Please only one space between the type and the variable name

Reply:
This was part of my PHPStorm-Problem but it is outdated.